stock_cycle_count/models/stock_location.py

```diff
@@ -20,19 +20,23 @@
 class StockLocation(models.Model):
     _inherit = 'stock.location'
 
-    @api.one
+    @api.multi
     def _compute_loc_accuracy(self):
-        history = self.env['stock.inventory'].search([
-            ('location_id', '=', self.id), ('state', '=', 'done')])
-        history = history.sorted(key=lambda r: r.write_date, reverse=True)
-        if history:
-            wh_id = self.get_warehouse(self)
-            wh = self.env['stock.warehouse'].browse(wh_id)
-            if len(history) > wh.counts_for_accuracy_qty:
-                self.loc_accuracy = mean(history[:wh.counts_for_accuracy_qty].
-                                         mapped('inventory_accuracy'))
-            else:
-                self.loc_accuracy = mean(history.mapped('inventory_accuracy'))
+        for rec in self:
+            history = self.env['stock.inventory'].search([
+                ('location_id', '=', rec.id), ('state', '=', 'done')])
+            history = history.sorted(key=lambda r: r.write_date, reverse=True)
```

Comment:
why not `order='write_date desc'` on search?

Reply:
mainly because this is not something flake8 / pylint flagged :smile_cat: The goal of the PR was originally to remove all the errors causing a red build and as many warnings as convenient. 

But I agree this one has to be fixed, and I just pushed the fix. Thanks